bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "paper-slider",
-  "version": "1.0.12",
+  "version": "1.0.13",
```

Comment:
Please don't update bower.json/version number with this PR - we will do that separately and publish a release separately.